content/pages/tools/jupyter-book.html

```diff
@@ -0,0 +1,137 @@
+{#
+  View this page at http://localhost:8080/tools/jupyter-book/
+#}
+<html>
+  <head>
+    <title>Publish your book on the internet with Read the Docs</title>
```

Comment:
```suggestion
    <title>Publish your Jupyter Book with Read the Docs</title>
```

Reply:
So:

- :+1: On mentioning `Jupyter Book with Read the Docs` in the title/h1
- But `Jupyter Book` is the tool, and https://jupyterbook.org/en/stable/intro.html uses `book` as the noun. Technically `Jupyter Book book` seems correct, but obviously sounds bad bad.

Instead, maybe this:

```suggestion
    <title>Publish your documentation with Jupyter Book and Read the Docs</title>
```